lib/resolveTaskFn.js

```diff
@@ -68,35 +68,36 @@ const makeErr = (command, result, ctx) => {
  *
  * @param {Object} options
  * @param {string} options.command — Linter task
+ * @param {string} [options.cwd]
  * @param {String} options.gitDir - Current git repo path
  * @param {Boolean} options.isFn - Whether the linter task is a function
  * @param {Array<string>} options.files — Filepaths to run the linter task against
- * @param {Boolean} [options.relative] — Whether the filepaths should be relative
```

Comment:
Should it be released as a breaking then?

Reply:
The `relative` option was actually unused, see here: 

https://github.com/okonet/lint-staged/blob/c7ea3594c81f7c2724a7babc8e8d57926b4679c8/lib/makeCmdTasks.js#L64

Reply:
...meaning the file paths are already relative to `cwd` when being passed here, but the `relative` flag itself wasn't doing anything.

Reply:
I think it's still worth indicating in the release notes that it was removed. And that is a breaking change IMO. Thoughts?

Reply:
It's not a breaking change because currently the `relative` option is never passed to the function. The link is from `master`, not this branch. This just removes it as "unused", and also fixes the behavior by always passing `cwd`.

Reply:
The relative option is still there, and controls whether file paths are absolute or not.

Reply:
What I mean is, I think it's currently broken so fixing it shouldn't be a "breaking change". 🤔

Reply:
I see. Okay thanks for explaining it to me :)